enrollment-server/src/main/java/com/wultra/app/enrollmentserver/controller/api/ConfigurationController.java

```diff
@@ -53,23 +47,11 @@ public ConfigurationController(final MobileApplicationConfigurationProperties mo
     /**
      * Fetch enrollment server configuration.
      *
-     * @param request request
-     * @param eciesContext ECIES context.
      * @return configuration
-     * @throws PowerAuthEncryptionException Thrown when request decryption fails.
      */
-    @PostMapping
+    @GetMapping
```

Comment:
Without encryption, we could switch to `GET`.

Reply:
Please no :) There is NO mobile `GET` endpoint and I'm not ready to accept one if it's just "we could" 😅

Reply:
Well, I'm not using it 🤣, also, the `GET` is not supported in our (read mine) networking SDK, so once we add a `GET` it should be for a very good reason, because for me, it's not just an annotation, but some coding for both platforms.

Reply:
I suggest we stick to `POST` because majority of our endpoints uses `POST` due to security requirements..

Reply:
I am aware this is not pure REST, however this is not the goal. We do support REST in PA cloud and other client-facing APIs, however having a clean REST API against mobile app is not what we aim for.

Reply:
No problem, reverted. I just wanted to make it consistent with onboarding endpoint, but as @kober32 has pointed out, that was not a good clue ;)